anchorecli/clients/apiexternal.py

```diff
@@ -85,7 +84,7 @@ def system_feeds_sync(config, flush=False):
     except Exception as err:
         raise err
 
-    return(ret)    
+    return(ret)
```

Comment:
this will need to get refactored to:

```suggestion
    return ret
```

Reply:
yeah lots of these in the code... the linter won't like our code much